integration/tests/tracer/test_tracer_debug_trace_complex_transactions.py

```diff
@@ -0,0 +1,213 @@
+        assert response["result"]["input"].lower() == instruction_tx["data"].lower()
+        assert response["result"]["type"] == "CALL"
+        assert "error" not in response["result"]
+
+    def test_trace_iterative_tx_sol_chain(self, web3_client_sol, class_account_sol_chain, counter_contract_sol_chain):
+        sender_account = class_account_sol_chain
+        tx = web3_client_sol.make_raw_tx(from_=sender_account)
+
+        instruction_tx = counter_contract_sol_chain.functions.moreInstruction(0, 3000).build_transaction(tx)
+        receipt = web3_client_sol.send_transaction(sender_account, instruction_tx)
+
+        wait_condition(
+            lambda: web3_client_sol.is_trx_iterative(receipt["transactionHash"].hex()) is True,
+            timeout_sec=120,
+        )
+
+        params = [receipt["transactionHash"].hex(), tracer_params]
+        response = self.tracer_api.send_rpc_and_wait_response("debug_traceTransaction", params)
+        assert response["result"]["from"].lower() == receipt["from"].lower()
+        assert response["result"]["to"].lower() == receipt["to"].lower()
+        assert response["result"]["input"].lower() == instruction_tx["data"].lower()
+        assert response["result"]["type"] == "CALL"
+        assert "error" not in response["result"]
+
+    def test_trace_iterative_tx_block_timestamp(self, block_timestamp_contract, json_rpc_client):
+        contract, _ = block_timestamp_contract
+        sender_account = self.accounts[0]
+
+        tx = self.web3_client.make_raw_tx(sender_account)
+        instruction_tx = contract.functions.callIterativeTrx().build_transaction(tx)
+        receipt = self.web3_client.send_transaction(sender_account, instruction_tx)
+        assert receipt["status"] == 1
+        assert self.web3_client.is_trx_iterative(receipt["transactionHash"].hex())
+
+        response = json_rpc_client.send_rpc(method="eth_getBlockByHash", params=[receipt["blockHash"].hex(), False])
+        tx_block_timestamp = EthGetBlockByHashResult(**response).result.timestamp
+
+        event_logs = contract.events.Result().process_receipt(receipt)
+        assert len(event_logs) == 1, "Event logs are not found"
+        assert event_logs[0]["args"]["block_timestamp"] <= int(tx_block_timestamp, 16)
+
+        params = [receipt["transactionHash"].hex(), tracer_params]
+        response = self.tracer_api.send_rpc_and_wait_response("debug_traceTransaction", params)
+        assert response["result"]["from"].lower() == receipt["from"].lower()
+        assert response["result"]["to"].lower() == receipt["to"].lower()
+        assert response["result"]["input"].lower() == instruction_tx["data"].lower()
+        assert response["result"]["type"] == "CALL"
+        assert "error" not in response["result"]
+
+    @pytest.mark.skip(reason="NDEV-3591")
+    def test_trace_scheduled_tx(self, web3_client_sol, neon_user, common_contract, evm_loader, treasury_pool):
```

Comment:
also please add a test or create a task for more tests for scheduled trx:
1) failed one
2) with multiple tree

Reply:
NDEV-3487

Reply:
I mean new one
to add more cases for scheduled trx

Reply:
sry, my bad, I actually created a new one and it is NDEV-3610 not 3487